src/test/java/pvs/app/entity/ProjectTest.java

```diff
@@ -0,0 +1,42 @@
+package pvs.app.entity;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+import pvs.app.Application;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.junit.Assert.assertEquals;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest(classes = Application.class)
+public class ProjectTest {
+    Project project;
+    Repository githubRepository;
+    Set<Repository> repositorySet;
+
+    @Before
+    public void setup() {
+        project = new Project();
+        project.setMemberId(1L);
+        project.setName("myProject");

```

Comment:
In order to ensure that the expected project name is the name of the test project, Suggest use a constant variable to declare this name and use it on both `project.setName` and `assertEquals`.
```java
final String testProjectName = "myProject";
```
```suggestion
        project.setName(testProjectName);
```